local/path/values.tmpl.yaml

```diff
@@ -19,16 +19,18 @@ guard:
   # 1. Each service must have a unique service ID.
   # 2. Each service can have a list of aliases that can be used to route requests to the service.
   services:
```

Comment:
Adding both `serviceId` and `aliases` with the same values is not needed as the `HTTPRoute` resources are created by making a new route for each of:
`serviceId + (aliases...)`

In my experience having duplicate entries for the same string values can also causes k8s issues in non-Tilt environments.

I suggest we do something like this:
```yaml
  services:
    - serviceId: eth
      aliases:
        - ethereum
    - serviceId: anvil
    - serviceId: pocket
    - serviceId: poly
      aliases:
        - polygon
```

This way we have examples for how aliases may be used (`eth/ethereum` and `poly/polygon`) while removing any redundant duplication of the same strings. IMO this makes the intended function of aliases clearer to users of PATH.